IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+}
+% page header for pages other than cover page
+\markboth{IREC Project Design Document}%
+{Mitchell \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+ The Intercollegiate Rocket Engineering Competition is a well-known competition among those interested in amateur rocket engineering as well as those
+ who do so professionally. With over 110 participating teams, it is the world's largest university rocket competition. Every year, more and more
+ organizations are getting involved such as the Space Dynamics Laboratory who sponsors the SDL Payload Challenge. By cooperatively partnering with
+ RIT Launch Initiative and their rocket-manufacturing experience, SPEX has an incredible opportunity to display engineering design proficiency and
+ an extreme passion for space by creating a custom scientific payload to integrate with Launch Initiative's custom rocket.
+
+
+      % The abstract is a brief summary of the design document. Typically it includes the purpose of the design document, key goals or objectives, and justifications.
+      % Be sure not to confuse the abstract with the introduction.
+      % It is easiest to write the abstract after the rest of the paper has been written.
+      % That way you can choose key information from the sections that you've already completed and string them together in the abstract.
+      % Consider the abstract to be your elevator pitch to anyone reading this design document.
+      % What are they reading?
+      % What is the goal?
+      % Why is it worth my time?
+      % The abstract is what will show up in Google results and other search engines, and what people will read when they are deciding what is worth their time and brain power.
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{
+    % If you include mathematical expressions or express variables in the design document, list them with their corresponding definitions here as a list.
+    % The two lines below make it look nice when defining units/values to constants.
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{LI}{Launch Initiative}
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{PDD}{Project Design Document}
+\nomenclature{IREC}{Intercollegiate Rocket Engineering Competition}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{SDL}{Space Dynamics Laboratory}
+\nomenclature{ESRA}{Experimental Sounding Rocket Association}
```

Comment:
Remove acronyms that do not appear in the body of the PDD, like "PDD".  Maybe also include SEDS here?

You can _either_ define acronyms in the nomenclature section OR define them in the body of the text (like "The Intercollegiate Rocket Engineering Competition (IREC)...") but not both. This is a style choice that I still don't ahve a firm stance on so use your own discretion